.husky/pre-commit

```diff
@@ -1,6 +1,3 @@
-#!/usr/bin/env sh
```

Comment:
was this intentional? want to make sure that the secrets scan is still being run

Reply:
Yeah this was intentional, during commit I notice husky was asking us to remove this from the script